atlp-rwanda/stackup2-barefoot-frontend
# #172990691 Landing Page

#### What does this PR do?

Add the landing page

#### Description of Task to be completed?
- Install required libraries
- Create NavBar component
- Update Home component
- Write tests

#### How should this be manually tested?
- In your terminal, run `git clone https://github.com/Stackup-Rwanda/stackup2-barefoot-frontend.git` to clone this repo

- Run `git fetch origin ft-landing-page-172990691:ft-landing-page-172990691` to pull this PR's branch

- Run `git checkout ft-landing-page-172990691` to switch to the fetched branch above

- Run `yarn install` to install all dependencies

- Start development server by running `yarn dev`.

- The server should launch the app in your default browser at `localhost:{{PORT}}`

#### Any background context you want to provide?
N/A
#### What are the relevant pivotal tracker stories?
  [#172990691](https://www.pivotaltracker.com/story/show/172990691)
